src/extensions/php_fpm_metrics_test.go

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package extensions
```

Comment:
in general I highly recommend keeping tests in a separate `*_tests` package.

From [The Zen of Go](https://the-zen-of-go.netlify.app/):

> *Write tests to lock in the behaviour of your package’s API*
>
> Test first or test later, if you shoot for 100% test coverage or are happy with less, regardless your package’s API is your contract with its users. Tests are the guarantees that those contracts are written in. Make sure you test for the behaviour that users can observe and rely on.

When our tests are in the same package as the code, we're able to use things outside the packages API, and potentially miss things.